src/index.js

```diff
@@ -8,6 +8,7 @@ const GLOBAL_CLIMATE_STRIKE_URLS = {
   de: 'https://de.globalclimatestrike.net/?source=digitalstrikebanner',
   cs: 'https://www.tydenproklima.cz',
   fr: 'https://fr.globalclimatestrike.net/?source=digitalstrikebanner',
+  nl: 'https://nl.globalclimatestrike.net/?source=digitalstrikebanner'
```

Comment:
Unfortunately Dutch isn't a supported language (yet) on the Global Climate Strike site, so this link will return a 404. I'm going to update it to be the English URL.